js/flightlog_parser.js

```diff
@@ -582,23 +582,21 @@ var FlightLogParser = function(logData) {
                 // Extract the firmware revision in case of Betaflight/Raceflight/Cleanfligh 2.x/Other
                 var matches = fieldValue.match(/(.*flight).* (\d+)\.(\d+)(\.(\d+))*/i);
                 if(matches!=null) {
-                
-                    // The that.sysConfig.firmwareType usually has been detected before that. Default value.
-                    if (typeof that.sysConfig.firmwareType === 'undefined' || !that.sysConfig.firmwareType) {
-                	    that.sysConfig.firmwareType = FIRMWARE_TYPE_BETAFLIGHT;                    
+
+                    // Detecting Betaflight requires looking at the revision string
+                    if (matches[1] === "Betaflight") {
```

Comment:
Indentation is wrong here. Make sure you consistently use 4 spaces and not tabs.